drivers/src/hand_off.rs

```diff
@@ -337,11 +342,12 @@ impl Default for FirmwareHandoffTable {
             ldevid_tbs_size: 0,
             fmcalias_tbs_size: 0,
             rtalias_tbs_size: 0,
-            reserved: [0u8; 126],
+            reserved: [0u8; 1566],
             ldevid_tbs_addr: 0,
             fmcalias_tbs_addr: 0,
             pcr_log_addr: 0,
             fuse_log_addr: 0,
+            ldevid_dice_sign: Ecc384Signature::default(),
```

Comment:
Probably for a separate PR, but we should re-order this Default to match the order of the actual structure. Better for clarity and maintenance

Reply:
opened https://github.com/chipsalliance/caliptra-sw/issues/769

Reply:
Ack, I can follow up on this